comparison.tex

```diff
@@ -0,0 +1,39 @@
+\section{Comparison}
+
+We present a few alternatives for performing proof of burn which have been proposed in previous work. All of the schemes presented are captured by our burn protocol definition.
+
+\newcommand{\opreturn}{\texttt{OP\_RETURN}}
+
+\noindent
+\textbf{$\opreturn$.}
+Bitcoin natively employs a mechanism for burning money using a specialized
+opcode called $\opreturn$~\cite{bartoletti2017analysis}. Unfortunately,
+creating an $\opreturn$ proof-of-burn transaction requires specialized software and thus is not
```

Comment:
"Unfortunately, creating an $\opreturn$ proof-of-burn transaction requires specialized software and thus is not user-friendly."
->
"Unfortunately, standard wallets do not provide a user-friendly interface for creating an $\opreturn$ proof-of-burn transaction."